x/protorev/keeper/posthandler.go

```diff
@@ -64,10 +65,10 @@ func (protoRevDec ProtoRevDecorator) PostHandle(ctx sdk.Context, tx sdk.Tx, simu
 		ctx.Logger().Error("ProtoRevTrade failed with error: " + err.Error())
 	}
 
-	// Delete swaps to backrun for next transaction without consuming gas
-	// from the current transaction's gas meter, but instead from a new gas meter with 50mil gas.
-	// 50 mil gas was chosen as an arbitrary large number to ensure deletion does not run out of gas.
-	protoRevDec.ProtoRevKeeper.DeleteSwapsToBackrun(ctx.WithGasMeter(sdk.NewGasMeter(sdk.Gas(50_000_000))))
+	// // Delete swaps to backrun for next transaction without consuming gas
+	// // from the current transaction's gas meter, but instead from a new gas meter with 50mil gas.
+	// // 50 mil gas was chosen as an arbitrary large number to ensure deletion does not run out of gas.
+	// protoRevDec.ProtoRevKeeper.DeleteSwapsToBackrun(ctx.WithGasMeter(sdk.NewGasMeter(sdk.Gas(50_000_000))))
```

Comment:
Don't we still need this? Just on the transient store?

Reply:
Do we? I thought the purpose of Transient stores was that it drops all states anyways so thought we wouldn't need to delete them 🤔

Reply:
It does drop all state, but at the end of the block not the end of each tx

Reply:
(notably these need to get deleted end of each tx, in order to not cause excess backruns next block)

Reply:
Ahh gotcha, confused myself with this logic being in endblocker. Fixed in https://github.com/osmosis-labs/osmosis/pull/7665/commits/46909089cf1f5ef16ad47823f272542b1eda7866 this commit!